src/nn/mod.rs

```diff
@@ -256,6 +259,7 @@ pub mod modules {
         TransformerEncoder, TransformerEncoderBlock,
     };
     pub use super::unbiased_linear::UnbiasedLinear;
+    pub use super::upscale::Upscale2D;
```

Comment:
You'll need to add to the builders mod below as well

Reply:
I did this for ConvTrans2D, but neither upscale has a builder.

Reply:
Ah yeah, but currently builders re-exports modules if there's no existing builder for it. The intent is someone can either do `use dfdx::modules::*` or `dfdx::builders::*` and get everything either way.